pkg/JuliaInterface/gap/adapter.gi

```diff
@@ -65,7 +65,7 @@ BindGlobal("_JL_RNAM_TO_JULIA_SYMBOL", function(rnam)
     if ISB_REC(_JL_RNAM_TO_JULIA_SYMBOL_CACHE, rnam) then
         symbol := ELM_REC(_JL_RNAM_TO_JULIA_SYMBOL_CACHE, rnam);
     else;
-        symbol := JuliaSymbol( NameRNam( rnam ) );
+        symbol := Julia.Symbol( NameRNam( rnam ) );  # TODO: use rnam -> symbol cache
```

Comment:
Concerning the caching idea:
Isn't this the `if` case, using the record `_JL_RNAM_TO_JULIA_SYMBOL_CACHE`?
Or would a Julia dictionary be more appropriate?

Reply:
Yes precisely -- those comments slipped in by accident (I made them during development then realized they are pointless and *thought* I had removed them... but clearly not *sigh*)